tests/CMakeLists.txt

```diff
@@ -1,15 +1,25 @@
 include(Catch)
 
-add_executable(tests test_config.cpp)
+set (TEST_PREFIX "test_${PROJECT_NAME}")
```

Comment:
Does this change need to be part of this review? I am specifically addressing the test name uniqueness in https://github.com/EVerest/everest-framework/pull/140

Reply:
I will remove this.